ext/mbstring/mbstring.c

```diff
@@ -1515,8 +1515,8 @@ PHP_FUNCTION(mb_http_output)
 	} else {
 		encoding = mbfl_name2encoding(name);
 		if (!encoding) {
-			php_error_docref(NULL, E_WARNING, "Unknown encoding \"%s\"", name);
-			RETURN_FALSE;
+			zend_argument_value_error(1, "must be a valid encoding, encoding \"%s\" is unknown", name);
```

Comment:
You now have multiple different error messages, part uses `"%s" given`, part uses `encoding "%s" is unknown`.

Reply:
Indeed, I thought I fixed them but apparently I didn't, will do this ASAP